docs/fieldnorm/verification_validation.md

```diff
@@ -1,7 +1,86 @@
 
-Donec id nibh aliquam, euismod velit id, aliquam ante. Vestibulum finibus interdum tellus quis efficitur. Quisque convallis nec nisl ut sollicitudin. Sed fringilla tellus et massa viverra interdum. Curabitur egestas diam at dui consectetur, et dapibus dui dignissim. Maecenas eget pulvinar enim. Etiam sodales id dui id venenatis. Duis id pellentesque tellus. Quisque enim justo, tristique id volutpat in, euismod ac tellus. Etiam faucibus velit sed purus commodo, eu vehicula velit egestas. Vivamus faucibus est et tortor imperdiet convallis. Vestibulum nulla enim, blandit a purus eget, cursus pellentesque lectus. Nulla finibus mi euismod eros lobortis, vitae eleifend erat rutrum.
\ No newline at end of file
+> Verification and Validation
+> The verification and validation (V&V) activities should be targeted towards the criticality and impact to patient safety of the SaMD, as discussed in IMDRF/SaMD WG/N12.Typically, verification (providing assurance that the design and development activity at each development stage conforms to the requirements) and validation (providing reasonable confidence that the software meets its intended use/user needs and operational requirements) activities ensure that all elements from the SaMD design and development—including any changes made during maintenance/upgrades—have been implemented correctly and that objective evidence of this implementation is recorded.
+
+In short, Barry Boehm expressed the difference between the software verification and software validation as follows:
+
+> Verification: Are we building the product right?
+> Validation: Are we building the right product?
+
+Hence, for each release _Software verification_ **and** _Software validation_ is required. 
+
+Often these definitions do not match with the definitions used in the medical laboratory. However, in terms of software development and thus bioinformatics, following software verification and software validation definitions make more sense. Noteworthy, the ISO-15189 states that validation is needed for laboratory designed or developed methods and for validated methods subsequently modified. Both are true for software development.
+
+Verification and Validation of software should be performed as described below. 
+
+The IEC-62304 mentiones different aspects of verification; i.e. software unit verification and software integration testing, and validation; i.e. software system test.
+
+
+
+
+## Software verification
+
+Testing whether the code is correctly developed can be done on different levels, for example through:
+
+- Code reviews 
+- Unit testing
+- Integration testing
+- End to end workflow tests
+
+The extend of software verification performed by the Bioinformaticians will differ in each team, it should be clear from the procedure which software verification should be performed. 
+
+## Software validation
+
+In order to support the repeatability of the validation, the following must be registered:
+
+- The person executing the test
```

Comment:
added: In case of automated testing, there must be a procedure of how the results are checked. Such as "a merge request is not possible when automated testing fails", passing of all automated testing could also be a note in the verification process.

Reply:
added: In case of automated testing, there must be a procedure of how the results are checked. I.e. a merge request is not possible when automated testing fails, or registering the passing of all automated testing among other verification results.

Reply:
> when the test is automated you do not need a report of that... you can make a note somewhere that all the automated tests passed

I agree, but this is not written in the fieldnorm. There should be clarity in this case.